src/TestFramework/Factory.php

```diff
@@ -142,11 +145,6 @@ public function create(string $adapterName, bool $skipCoverage): AbstractTestFra
             );
         }
 
-        throw new \InvalidArgumentException(
-            sprintf(
-                'Invalid name of test framework. Available names are: %s',
-                implode(', ', [TestFrameworkTypes::PHPUNIT, TestFrameworkTypes::PHPSPEC])
-            )
-        );
+        throw new \InvalidArgumentException(sprintf('Invalid name of test framework "%s". Available names are: %s', $adapterName, implode(', ', [TestFrameworkTypes::PHPUNIT, TestFrameworkTypes::PHPSPEC])));
```

Comment:
I guess you ran `make cs` before the fix was merged. Let's revert it - it's not readable now.

Reply:
arg indeed